extensions/wikia/AdEngine/js/spec/lookup/prebid/adapters/openx.spec.js

```diff
@@ -0,0 +1,55 @@
+/*global describe, expect, it, jasmine, modules*/
+describe('ext.wikia.adEngine.lookup.prebid.adapters.openx', function () {
+	'use strict';
+
+	var mocks = {
+		instantGlobals: {
+			wgAdDriverOpenXPrebidBidderCountries: ['PL']
+		},
+		geo: {
+			isProperGeo: jasmine.createSpy('isProperGeo')
```

Comment:
nitpick: if we use it only in one place, could we use `spyOn(mocks. geo, 'isProperGeo');` in 'isEnabled checks the countries instant global' test case?